docs/organization-integration/usermanagement.md

```diff
@@ -89,17 +89,84 @@ SIG will setup SSO for you. You will have your own customer-specific URL Sigrid.
+
+Similarly, while Maintainers enjoy additional admin-level permissions on those systems that are accessible to them, there are still limitations on specific actions that are reserved for Administrator users only. This includes the following: 
+- Maintainer users cannot authorize themselves for new systems
+- Maintainer users cannot modify or create portfolio-level objectives
+- Maintainer users cannot grant the Admin role to any user, themselves included
+- Maintainer users cannot update access rights or user details for Administrator users, nor are they able to adjust the system access rights of other users for systems they themselves do not have access to
+- Maintainer users cannot create or delete users or authorization groups
+
+### Authorized Actions based on User Type
+
+For a full breakdown of tasks able to be performed by each user type, please refer to the following authorized actions table. *Note:* These actions refer to both manual actions within the Sigrid web interface as well as actions allowed [via the Sigrid API](../integrations/sigrid-api-documentation.md#managing-user-permissions-via-API):
+
+| Action | Normal User | Maintainer | Administrator |
+| --- | --- | --- | --- |
+| **Portfolio Overview**
+| View Dashboard | ✔ | ✔ | ✔✔ | 
+| View Systems | ✔ | ✔ | ✔✔|
+| **Capability Specific Overviews**
+| View Dashboards | ✔ | ✔ | ✔✔ |
+| View Ratings | ✔ | ✔ | ✔✔ |
+| Favorite Systems | ✔ | ✔ | ✔✔ |
+| **Findings and Refactoring Candidates**
+| View Findings | ✔ | ✔ | ✔✔ |
+| Edit Findings | ✔ | ✔ | ✔✔ |
+| View Finding Audit Trails | ✔ | ✔ | ✔✔ |
+| Create Manual Findings |  | ✔ | ✔✔ |
+| **Source Code**
+| View Source Code | ✔ | ✔ | ✔✔ |
+| **Meta-data** 
+| View metadata | ✔ | ✔ | ✔✔ |
+| Configure metadata |  | ✔ | ✔✔ |
+| **Objectives**
+| View objectives | ✔ | ✔ | ✔✔ |
+| Configure system-level objectives |  | ✔ | ✔✔ |
+| Configure portfolio-level objectives |  |  | ✔✔ |
+| **User Management**
+| View Users | ✔ | ✔ | ✔✔ |
+| Create / Delete Users |  |  | ✔✔ |
+| Edit User Permissions |  | ✔ | ✔✔ |
+| Edit User Details (other than self) |  |  | ✔✔ |
+| View User Status Details |  |  | ✔✔ |
+| Edit User Role |  |  | ✔✔ |
+| Reset User Password / MFA |  |  | ✔✔ |
+| Create / Delete Authorization Groups |  |  | ✔✔ |
+| Edit Authorization Groups |  |  | ✔✔ |
+
+✔ **Limited Scope:** User is only able to perform this action on systems that are explicitly accessible to the user
+
+✔✔ **Global Scope**: User is able to perform this action across any system in the portfolio
```

Comment:
For the sake of readability in dark mode, I propose to replace the ✔s with ✅s. Also split up the Edit Authorization Groups in its 3 sub-parts.

```suggestion
| **Portfolio Overview**
| View Dashboard | ✅ | ✅ | ✅✅ |
| View Systems | ✅ | ✅ | ✅✅ |
| **Capability Specific Overviews**
| View Dashboards | ✅ | ✅ | ✅✅ |
| View Ratings | ✅ | ✅ | ✅✅ |
| Favorite Systems | ✅ | ✅ | ✅✅ |
| **Findings and Refactoring Candidates**
| View Findings | ✅ | ✅ | ✅✅ |
| Edit Findings | ✅ | ✅ | ✅✅ |
| View Finding Audit Trails | ✅ | ✅ | ✅✅ |
| Create Manual Findings | | ✅ | ✅✅ |
| **Source Code**
| View Source Code | ✅ | ✅ | ✅✅ |
| **Meta-data**
| View metadata | ✅ | ✅ | ✅✅ |
| Configure metadata | | ✅ | ✅✅ |
| **Objectives**
| View objectives | ✅ | ✅ | ✅✅ |
| Configure system-level objectives | | ✅ | ✅✅ |
| Configure portfolio-level objectives | | | ✅✅ |
| **User Management**
| View Users | ✅ | ✅ | ✅✅ |
| Create / Delete Users | | | ✅✅ |
| Edit User Permissions | | ✅ | ✅✅ |
| Edit User Details (other than self) | | | ✅✅ |
| View User Status Details | | | ✅✅ |
| Edit User Role | | | ✅✅ |
| Reset User Password / MFA | | | ✅✅ |
| Create / Delete Authorization Groups | | | ✅✅ |
| Edit Authorization Group Permissions | | ✅ | ✅✅ |
| Edit Authorization Group Members | | ✅(1) | ✅✅ |
| Edit Authorization Group Details | | ✅(1) | ✅✅ |

✅ **Limited Scope:** User is only able to perform this action on systems that are explicitly accessible to the user  
✅✅ **Global Scope**: User is able to perform this action across any system in the portfolio  
(1) Only for Groups the Maintainer is a member of
```